README.md

```diff
@@ -1,11 +1,46 @@
-concourse-dcind
-===============
+pages-images
+============
 
-A Concourse CI resources type to run Docker Compose in Docker
+A series of common container images used in the cloud.gov Pages system.
 
 ## Why this project
 
-Systems, apps, and tools developed and tested using Docker Compose can also be run using Docker Compose from within a Concourse CI task.
+Pages relies on Concourse CI and CloudFoundry Apps to deliver and serve the platform's different components. This is a centralized repository to hold all of the commonly used images which are hardened and scanned using the cloud.gov [common-pipelines](https://github.com/cloud-gov/common-pipelines) container hardening CI framework.
+
+## How this works
+
+Create a directory in the root of the repository for the name of the image and then a subdirectory for the images's version. To create a Node v20 image, create the directories `./node/v20` and then add the `Dockerfile` and any other supporting scripts under the image version directory.
+
+All Dockerfiles will take the `ARG base_image` argument which is a hardened Ubuntu Jammy image based the [common-pipelines OCI Pipeline](https://github.com/cloud-gov/common-pipelines/blob/main/container/README.md).
+
+### Registering a new image
+
+To add a new image to `pages-images` you will also need to create a new ECR repository for the common pipeline image put step to use. Add the name of the repository to the [repositories list](https://github.com/cloud-gov/cg-provision/blob/main/terraform/stacks/ecr/stack.tf) in Terraform and make sure the update has been applied.
+
+### Running Locally
+
+To get up and running quickly, you can build an image locally using the Ubuntu Jammy (v22.04) image as your `base_image`.
+
+```sh
+# Navigate into the directory of the image and version you want to build
+# Use the general Ubuntu Jammy base image for local development
+docker buildx build . -t <image-tag> --build-arg base_image=ubuntu:22.04
```

Comment:
Is there any reason to tag these builds?

Reply:
To use the image again locally.